modules/apps/portal-settings/portal-settings-web/src/META-INF/resources/init.jsp

```diff
@@ -14,7 +14,7 @@
  */
 --%>
 
-<%@ include file="/html/portlet/init.jsp" %>
+<%@ include file="/portlet/init.jsp" %>
```

Comment:
this file doesn't exist, nor should it. Can remove this line and embed the parts you need from that file in core.